### PR TITLE
feat(NA): Add support for custom metrics.

### PR DESCRIPTION
**Pull request by reistiago**
Endpoint exposed on the event bus to consume custom metrics.

This is missing documentation, and proper refactor of some parts of the code. Done quickly to support one of our clients

### CODE REVIEW

File: src/main/java/com/statful/client/CustomMetricsConsumer.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package com.statful.client;
+
+import com.statful.sender.Sender;
+import io.vertx.core.eventbus.EventBus;
+
+/**
+ * Allows consumers to send custom metrics thought the event bus
```
Comment (brunopalos) [inline src/main/java/com/statful/client/CustomMetricsConsumer.java]: Isn't this a typo? `thought`? Shouldn't be something like `sent thought`? :)

**Review by brunopalos — APPROVED** [review]
lgtm

File: src/main/java/com/statful/client/CustomMetricsConsumer.java (hunk 1 above)
Comment (reistiago) [reply src/main/java/com/statful/client/CustomMetricsConsumer.java]: typo yes, it should be "through".

Thank you

File: src/main/java/com/statful/client/MetricType.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    COUNTER("counter"),
+    /**
+     * Type for timer metrics
+     */
+    TIMER("timer");
+
+    /**
+     * String representation of this metric type
+     */
+    private String value;
```
Comment (miguelcnf) [inline src/main/java/com/statful/client/MetricType.java:23]: Final?

**Review by miguelcnf — APPROVED** [review]

**Comment by miguelcnf** [comment]
Released `1.0.0-alpha-8` to maven central.

@reistiago can you just add some docs on usage to the README when you have the time?

**Pull request merged**